src/styles/index.css

```diff
@@ -1,5 +1,10 @@
-/* Shift global.css here */
+@import "https://rsms.me/inter/inter.css";
```

Comment:
Definitely add new imports to PR description. 
I'm guessing this has something to do with the font ?

Reply:
Yep, I didn't add it because it was part of the old `global.css` as well, so I thought it was implied when saying 'migrated'. My bad.